prebid_cache_client/client_test.go

```diff
@@ -113,6 +117,45 @@ func TestEncodeValueToBuffer(t *testing.T) {
 	assertStringEqual(t, expected, actual)
 }
 
+// The following test asserts that the cache client's GetExtCacheData() implementation is able to pull return the exact Path and Host that were
+// specified in Prebid-Server's configuration, no substitutions nor default values.
+func TestStripCacheHostAndPath(t *testing.T) {
+	type aTest struct {
+		inConfig     []byte
+		expectedHost string
+		expectedPath string
+	}
+	testInput := []aTest{
+		{inConfig: []byte(`
+external_cache:
+  host: prebid-server.prebid.org
+  path: pbcache/endpoint
+`), expectedHost: "prebid-server.prebid.org", expectedPath: "pbcache/endpoint"},
+		{inConfig: []byte(`
+external_cache:
+  host: prebidcache.net
+`), expectedHost: "prebidcache.net", expectedPath: ""},
+		{inConfig: []byte(``), expectedHost: "", expectedPath: ""},
+	}
+	for i, test := range testInput {
+		//start viper
+		v := viper.New()
```

Comment:
Only recommendation I have is that here, you don't need to setup Viper and do a `ReadConfig` to read the config and then pass it on to the `NewClient` function because we're already testing Viper's capability to be able to do that properly in `config_test.go` file. Here, we just want to test the `NewClient` function and so we can directly create a `Cache` obj and `ExtCacheURL` object and pass them to `NewClient`

Reply:
Removed Mansi